CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKDevice.java

```diff
@@ -171,22 +180,22 @@ private int InitVKPhysicalDevice(MemoryStack stack) {
         
         // Get the number of physical devices
         IntBuffer pInt = stack.mallocInt(1);
-        checkVKret(vkEnumeratePhysicalDevices(cvkInstance.GetInstance(), pInt, null));
+        checkVKret(vkEnumeratePhysicalDevices(cvkInstance.GetVkInstance(), pInt, null));
         if (pInt.get(0) == 0) {
             throw new RuntimeException("Vulkan: no GPUs found");
         }
 
         // Get the physical devices
         int numDevices = pInt.get(0);
         PointerBuffer physicalDevices = stack.mallocPointer(numDevices);
-        ret = vkEnumeratePhysicalDevices(cvkInstance.GetInstance(), pInt, physicalDevices);
+        ret = vkEnumeratePhysicalDevices(cvkInstance.GetVkInstance(), pInt, physicalDevices);
         if (VkFailed(ret)) return ret;
```

Comment:
Not a line I changed in this CL so any change will happen later.  That said I think it needs braces but for just this use case I think a single line is ok.  The reason it started I think was replacing checkVKret at runtime and not wanting the line numbers to go out of whack before the next recompile.  There are a lot of them so it would be a bit of time for no gain imo.